tests/parm/fd_ufs.yaml

```diff
@@ -92,6 +92,7 @@
        description: atmosphere export
      #
      - standard_name: Faxa_rainl
+       alias: mean_prec_rate_large
```

Comment:
@uturuncoglu Thanks for making all the updates so far. 

I'm confused about these three aliases though for rainl, snowc and snowl. These are CDEPS-ATM exports, correct? So we're using CESM-style field names and I don't think the aliases are required. 

Could you also give me an update on your testing status? You've waited so long for this PR, I'd like to make sure it gets scheduled quickly. Thanks.

Reply:
@DeniseWorthen As I remember those aliases are the ones that are used in FV3 side but I could double check and get back to you. Thanks for all your help and review.

Yes. It would be great if we could get this PR soon. Then, I could use the recent version in AMS tutorial and also mentioned about it in the JTTI progress report which has deadline end of this month.

I run the all RTs and they were fine except aoflux issue that we fixed in the CMEPS side. Then, I run couple of aoflux tests and they were passing too. I think this is ready in terms of testing but if you want I could try to run again from scratch again on Orion with Intel. BTW, I am not sure CMEPS and CDEPS PR needs to go first ESCOMP or not. At this point, they are pointing to NOAA-EMC fork but maybe I need to point ESCOMP and then update NOAA-EMC. Anyway, let me know your preference.

Reply:
@DeniseWorthen Okay. I check the fields and they can be provided by CDEPS (ERA5 data atmosphere mode). At this point, you could provide large and convective parts separately or you could combine them together as total precipitation. You are right, we might not need to have aliases for those fields but to follow the convention with other fields such as Faxa_rain, I add them. If you want I could remove them test again.

Reply:
We're not actually sending those from the FV3-ATM right now and if we were to do so in the future, we'd want to use inst.

I wasn't sure about the ESCOMP issue---were you able to get CESM tests running on derecho? I think it is also perfectly acceptable to go to EMC first and then we can push back. I think the issue of CESM testing is more critical for the in-line functionality.

Reply:
@DeniseWorthen I could update the aliases by replacing mean with inst if you want. I could run CESM test on Derecho but there are lots of issue in there not related with the PR. So, it would be nice to push them NOAA-EMC fork first. Then, I could update my inline PR in ESCOM side to push those maybe.

Reply:
Yes, on please change the aliases to inst. I really want to move away from any "mean" fields being sent to CMEPS.

I think pushing to EMC first makes sense. Even the inline could go to EMC first, although I'd feel a lot better if it got tested in CESM before we did that.

Reply:
@DeniseWorthen Okay. I changed them to be inst. BTW, I realized that there are still filed in the yaml file start with mean but I think we are not aiming to fix all in this PR. That could be another one.

Reply:
Thanks. We will tackle to 'mean' aliases in the future.